### ISSUE TITLE
How to transition to the npm package?

### ISSUE THREAD

**Issue by nickredmark** [MeteorPackaging/Font-Awesome#15]
Has someone transitioned successfully to the npm package? What do you need to import and how?

**Comment by GenZai** [comment]
@nmaro I'm also wondering how to use npm instead of this package.
Have you found a solution ?

**Comment by nickredmark** [comment]
@GenZai nope, this is one of the few meteor packages I still depend on. I'm slowly working to replace the full meteor stack (with graphql and next.js), but it's a long way - if you're interested I'm mainly working on the user accounts side with a library called [ooth](https://github.com/nmaro/ooth).

**Comment by lmaddio** [comment]
I did a install with `meteor npm install font-awesome` and then I imported the .scss files, but is not working, says it doesn't find the file. Is strange because is the same way I import bootstrap:
`@import "{}/node_modules/font-awesome/scss/font-awesome.scss";`
`@import "{}/node_modules/bootstrap/scss/bootstrap.scss";`
